seismic/xcorqc/correlator.py

```diff
@@ -313,6 +304,14 @@ def outputConfigParameters():
               help="FDSNxml inventory containing instrument response information. Note that when this parameter is provided "
                    ", instrument response corrections are automatically applied for matching stations with response "
                    "information.")
+@click.option('--instrument-response-output',
```

Comment:
Consider adding `show_default=True` option to click options where there is a default value, to make the help text more informative to the user.